### PR TITLE
bugfix: NullPointerException in DB mock during CI

### PR DESCRIPTION
**Pull request by l81893521**
### Ⅰ. Describe what this PR did
When set the mock value null, would cause some NullPointerException during the CI.

### Ⅱ. Does this pull request fix one issue?
#1916

### CODE REVIEW

File: rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java [hunk 1]
```diff
@@ -110,6 +110,9 @@ public MockPreparedStatement createSeataMockPreparedStatement(MockConnection con
      * @param mockReturnValue
      */
     public void setMockReturnValue(Object[][] mockReturnValue) {
+        if (mockReturnValue == null) {
+            this.mockReturnValue = new Object[][]{};
```
Comment (helloworlde) [inline rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java]: Need return here, change `if ... else ...` or `... ? ... : ...` could be better

File: rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java [hunk 2]
```diff
@@ -134,6 +141,9 @@ public void setMockColumnsMetasReturnValue(Object[][] mockColumnsMetasReturnValu
      * @param mockIndexMetasReturnValue
      */
     public void setMockIndexMetasReturnValue(Object[][] mockIndexMetasReturnValue) {
+        if (mockIndexMetasReturnValue == null) {
+            this.mockIndexMetasReturnValue = new Object[][]{};
```
Comment (helloworlde) [inline rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java]: Same as above.

File: rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java (hunk 1 above)
Comment (l81893521) [reply rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java]: Fixed. Thanks

File: rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java (hunk 2 above)
Comment (l81893521) [reply rm-datasource/src/test/java/io/seata/rm/datasource/mock/MockDriver.java]: Fixed. Thanks

**Pull request merged**